src/layout/Border.js

```diff
@@ -51,6 +51,28 @@
 
     Border.prototype.publish("sectionTypes", [], "array", "Section Types sharing an index with 'content' - Used to determine position/size.", null, { tags: ["Private"] });
 
+    Border.prototype.watchWidget = function (widget) {
+        if(this._watch === undefined){
+            this._watch = {};
+        }
+        if (this._watch[widget.id()]) {
+            this._watch[widget.id()].remove();
+            delete this._watch[widget.id()];
+        }
+        if (widget) {
+            var context = this;
+            this._watch[widget.id()] = widget.monitor(function (paramId, newVal, oldVal) {
+                if (oldVal !== newVal) {
+                    context.lazyRender();
+                }
+            });
+        }
+    };
+    
+    Border.prototype.lazyRender = Border.prototype.debounce(function(){
+        this.postUpdate();
+    },0);
```

Comment:
You will want a debounce timeout > 0 otherwise the debounce doesn't do its thing...